blocks/DataOutput/DataOutput.js

```diff
@@ -22,63 +22,71 @@ export class DataOutput extends UploaderBlock {
 
   initCallback() {
     super.initCallback();
-    this.sub('output', (data) => {
-      if (!data) {
-        return;
-      }
+    this.sub(
+      'output',
+      (data) => {
+        if (!data) {
```

Comment:
May be?
```
!data || data?.length === 0
```

Reply:
Empty array is also valid value for output.